manifests/server.pp

```diff
@@ -76,6 +76,7 @@
   Boolean $create_admin                                 = $mongodb::params::create_admin,
   String $admin_username                                = $mongodb::params::admin_username,
   Optional[String] $admin_password                      = undef,
+  Optional[String] $admin_password_hash                 = undef,
```

Comment:
```suggestion
  Optional[String[1]] $admin_password_hash              = undef,
```

I think setting the hash to an empty string doesn't make much sense

Reply:
Done :)